drivers/SmartThings/zwave-switch/src/zooz-zen-30-dimmer-relay/init.lua

```diff
@@ -102,25 +164,15 @@ local function central_scene_notification_handler(driver, device, cmd)
   end
 end
 
-local function added_handler(driver, device)
-  device:emit_event(capabilities.button.supportedButtonValues(BUTTON_VALUES, { visibility = { displayed = false }}))
-  device:emit_event(capabilities.button.numberOfButtons({ value = 3 }, { visibility = { displayed = false }}))
-end
-
-local function do_refresh(driver, device)
-  device:send_to_component(SwitchMultilevel:Get({}), "main")
-  device:send_to_component(SwitchBinary:Get({}), "switch1")
-  device:send(Version:Get({}))
-end
-
 local function switch_set_on_off_handler(value)
   return function(driver, device, command)
+    local ep = device:component_to_endpoint(command.component)[1]
     local get, set
 
-    if command.component == "main" then
+    if ep == ENDPOINTS.dimmer then
       set = SwitchMultilevel:Set({ value = value, duration = constants.DEFAULT_DIMMING_DURATION })
       get = SwitchMultilevel:Get({})
-    elseif command.component == "switch1" then
+    elseif ep == ENDPOINTS.relay then
       set = SwitchBinary:Set({ target_value = value, duration = 0 })
       get = SwitchBinary:Get({})
     end
```

Comment:
I'd use capabilities again here. Using the endpoint is confusing to read.

Reply:
Done